backends/btor/test_cells.sh

```diff
@@ -6,7 +6,7 @@ rm -rf test_cells.tmp
 mkdir -p test_cells.tmp
 cd test_cells.tmp
 
-../../../yosys -p 'test_cell -n 5 -w test all /$alu /$fa /$lcu /$lut /$sop /$macc /$mul /$div /$mod /$divfloor /$modfloor'
+../../../yosys -p 'test_cell -n 5 -w test all /$alu /$fa /$lcu /$lut /$sop /$macc /$mul /$div /$mod /$divfloor /$modfloor /$shiftx'
```

Comment:
Could you explain the rationale for changes to this file?

Reply:
As far as I can tell, there is no way to test `$shiftx` with the method used in this file. The `gate` module is passed through `synth`, which collapses all the possible `x` outputs to either `0` or `1`. Meanwhile, the BTOR model for `gold` still produces `x` outputs, which are modeled as additional BTOR `input`s. Therefore, it is always possible to find some combination of BTOR inputs that collapses the undefs differently to how `synth` did it, resulting in a test failure.

`miter -equiv -ignore_gold_x` doesn't help either, because `== x` is just modeled as a comparison with another new input that can take on whatever value.